tools/lib/capitalization.py

```diff
@@ -74,6 +74,7 @@
     # SPECIAL CASES
     # Because topics usually are lower-case, this would look weird if it were capitalized
     r"more topics",
+    r"more conversations",
```

Comment:
@timabbott , @punchagan  -- I added this in `capitalization.py` so as to allow to write `more conversations` similar to what we have for `more topics`, So would like to ask for a suggestion for adding a comment above this added line which can make it easier for others to understand.

Reply:
I'd do:

> # We want the similar text in the Private Messages section to have the same capitalization.

Reply:
Thank you for this!

Reply:
Updated the branch.